miqi/sandbox/manager.py

```
@@ -307,6 +307,9 @@ def describe_exec_environment(
             "cmd 语法注意：用 && 连接多条命令（不支持 ; 分隔），"
             "ls/find/grep/sed 不可用（用 dir / where / findstr），"
             "或使用 powershell -Command \"...\"。"
+            "本机未安装 Git Bash 与 Windows 子系统，请勿运行 bash/wsl 命令"
```

Ruff (0.16.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)